### DISCUSSION TITLE
Restore sleeps after reinstall

### DISCUSSION THREAD

**Discussion by omnigenous** [vmiklos/plees-tracker#446] [Q&A] [answered]
I have uninstall the app because of: https://github.com/vmiklos/plees-tracker/discussions/445, now installed it back and my sleeps are gone. Is there a way to restore them? I don't think I had auto backup enabled :(

**Comment by vmiklos** [comment, marked as answer]
Uh-oh. One downside of the lack of big brother cloud backup is that no, if you deleted your sleeps database and had no backup, then you get it back.

I'm sorry the versioning change confused you, I promise not to change to a different scheme every year. :-)

**Reply by omnigenous** [reply]
Was in panic mode because of lack of sleep (ironic lol) and thought config and database don't get wipe on uninstall. That's what I'm used to from linux.

**Reply by vmiklos** [reply]
Ah yes, the Android default behavior is different, and I'm not sure an app can request keeping its data on uninstall, it's probably considered a feature that you can reliably uninstall an app and its config. :-)

Indeed, best to backup your sleeps before uninstall, and if you think you would forget that, then setting up automatic backup is also an option.
